src/main/kotlin/com/devfloor/untitled/hashtag/domain/HashtagRepository.kt

```diff
@@ -2,4 +2,6 @@ package com.devfloor.untitled.hashtag.domain
 
 import org.springframework.data.jpa.repository.JpaRepository
 
-interface HashtagRepository : JpaRepository<Hashtag, Long>
+interface HashtagRepository : JpaRepository<Hashtag, Long>{
+    fun findByNameIn(hashtagNameList: List <String>): List<Hashtag>
+}
```

Comment:
여기도 반영 안 된 부분 같아요 ! 이 부분에 대해서는 Unresolve conversation 누르셔서 리뷰 내용 한번 확인하시고 반영하시면 좋을 것 같아요!
```suggestion
interface HashtagRepository : JpaRepository<Hashtag, Long>{
    fun findAllByNameIn(hashtagNames: List <String>): List<Hashtag>
}
```